README.md

```diff
@@ -14,12 +14,20 @@ Spotipy's full documentation is online at [Spotipy Documentation](http://spotipy
 pip install spotipy
 ```
 
+alternatively 
+
+```bash
+py -m pip install spotipy
+```
+
 or upgrade
 
 ```bash
 pip install spotipy --upgrade
 ```
 
+After installation if your IDE can not find the libary, add an intepretor path to the library in your IDE. 
```

Comment:
this is not spotipy related. One is expected to be able to set up python libraries before reading this readme

Reply:
That makes sense. I  ran into problems when I successfully installed the library but ran into ModuleNotFoundError when trying to use it which took a lot of internet searching to resolve. I don't have a lot of experience installing libraries.